README.md

```diff
@@ -103,6 +103,23 @@ function write() {
 }
 ```
 
+### Setup and write to a pin that starts as on
```

Comment:
Give an example that uses the DIR_HIGH setting for direction. Didn't bother making a DIR_LOW one because it would be so similar.